### ISSUE TITLE
Mesh

### ISSUE THREAD

**Issue by jhefer123** [tschaban/SONOFF-firmware#23]
Hi

Would it be possible to add a mesh capability? 

Regards,
Johan

**Comment by tschaban** [comment]
do you mean each sonoff connects to the other and transfer data packages between each other to allow coverage wider than WiFi coverage - like in z-wave devices?

**Comment by jhefer123** [comment]
yes exactly that, i have read that its possible with the esp to do a mesh config, so lets say if one of my devices is in wifi coverage but the other is on the edge but close to the second device it would if it loses wifi coverage look for a nearby device and connect through it to the network, so if you setup the devices you set them up for all to connect to the wifi first preference but in the case of wifi signal dropping they would see if there is a nearby device they could connect to and also if they use each other and a link goes down they would auto reroute https://www.google.co.za/imgres?imgurl=http%3A%2F%2Fespressif.com%2Fsites%2Fdefault%2Ffiles%2Fwebform%2Fresume%2Fpastedgraphic-3.jpg&imgrefurl=http%3A%2F%2Fespressif.com%2Fen%2Fproducts%2Fsoftware%2Fesp-mesh%2Foverview&docid=UogdbddW-wNisM&tbnid=Lv1QvZ-PYBWA2M%3A&vet=10ahUKEwj9r--4vv7VAhVnD8AKHQ8aCWgQMwgoKAMwAw..i&w=891&h=381&bih=950&biw=1920&q=mesh%20network%20esp8266&ved=0ahUKEwj9r--4vv7VAhVnD8AKHQ8aCWgQMwgoKAMwAw&iact=mrc&uact=8
